cli/helpers.js

```diff
@@ -289,6 +289,16 @@ function parseCommandLineArguments() {
             type: "string",
             describe: "Title for the output HTML page.",
             default: "Protected Page",
+        })
+        .option("color-primary", {
```

Comment:
The CLI options [will change](https://github.com/robinmoisson/staticrypt/issues/166) in v3 to be more consistent, all options related to the template being prefixed by `--template`. We can start using that convention right away with `template-color-primary/secondary`